src/sentry/static/sentry/app/components/assistant/guideAnchor.tsx

```diff
@@ -116,13 +117,59 @@ const GuideAnchor = createReactClass<Props, State>({
     dismissGuide(currentGuide.guide, step, orgId);
   },
 
-  render() {
+  getHovercardExpBody() {
+    const {currentGuide, step} = this.state;
+
+    const totalStepCount = currentGuide.steps.length;
+    const currentStepCount = step + 1;
+    const currentStep = currentGuide.steps[step];
+    const lastStep = currentStepCount === totalStepCount;
+    const hasManySteps = totalStepCount > 1;
+
+    return (
+      <GuideExpContainer>
+        <GuideContent>
+          <GuideTitle>{currentStep.title}</GuideTitle>
+          <GuideDescription>{currentStep.description}</GuideDescription>
+        </GuideContent>
+        <GuideAction>
+          {lastStep ? (
+            <div>
```

Comment:
Is this div necessary?

Reply:
the button group needs a div for css grid, but moved it up